src/Umbraco.Web.UI/Controllers/AccountSurfaceController.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Umbraco.Web.UI.Controllers
+{
+    using System.Web.Mvc;
+    using System.Web.Security;
+    using Mvc;
+
+    public class AccountSurfaceController : SurfaceController
```

Comment:
Yes, looks like I was testing something out.  Away myself at the moment but if you don't do anything further with this by the end of the week I'll clean this up.